tslint.json

```diff
@@ -1,51 +1,69 @@
 {
 	"rules": {
+		"align": false,
 		"ban": [],
 		"class-name": true,
 		"comment-format": [ true, "check-space" ],
 		"curly": true,
 		"eofline": true,
 		"forin": false,
-		"interface-name": false,
+		"indent": [ true, "tabs" ],
+		"interface-name": [ true, "never-prefix" ],
 		"jsdoc-format": true,
 		"label-position": true,
 		"label-undefined": true,
-		"max-line-length": false,
+		"max-line-length": 120,
+		"member-access": false,
+		"member-ordering": false,
 		"no-any": false,
 		"no-arg": true,
 		"no-bitwise": false,
-		"no-console": false,
 		"no-consecutive-blank-lines": true,
+		"no-console": false,
 		"no-construct": false,
+		"no-constructor-vars": true,
 		"no-debugger": true,
 		"no-duplicate-key": true,
```

Comment:
FYI this will be removed in the next major bump of tslint, might as well remove it now